extensions/flags/tests/integration/api/flags/ListTest.php

```diff
@@ -77,12 +78,14 @@ public function admin_can_see_one_flag_per_post()
             ])
         );
 
-        $this->assertEquals(200, $response->getStatusCode());
+        $body = $response->getBody()->getContents();
 
-        $data = json_decode($response->getBody()->getContents(), true)['data'];
+        $this->assertEquals(200, $response->getStatusCode(), $body);
+
+        $data = json_decode($body, true)['data'];
 
         $ids = Arr::pluck($data, 'id');
```

Comment:
This variable is now unused. Is the problem that sort order is no longer deterministic?

Reply:
When grouping by, PgSQL acts differently than sqlite and mysql. PgSQL picks the first element of a group, while the others pick the last. That shouldn't be a problem since all we need is that it picks one flag only from each post regardless of which it is.

I've changed this to:

```php
        if ($this->database() instanceof PostgresConnection) {
            $this->assertEqualsCanonicalizing(['3', '4', '5'], $ids);
        } else {
            $this->assertEqualsCanonicalizing(['1', '4', '5'], $ids);
        }
```